tests/page-objects/common/common.po.js

```diff
@@ -112,18 +119,33 @@ module.exports = {
   },
 
   goToMessages: () => {
+    utils.deprecated('common.po.goToMessages', 'commmon.po.goToMessagesNative');
     browser.get(utils.getBaseUrl() + 'messages/');
     helper.waitUntilReady(medicLogo);
     helper.waitUntilReady(element(by.id('message-list')));
   },
 
+  goToMessagesNative: async () => {
+    await browser.get(utils.getBaseUrl() + 'messages/');
+    await helper.waitUntilReady(medicLogo);
+    await helper.waitUntilReady(element(by.id('message-list')));
+  },
+
   goToPeople: async () => {
+    utils.deprecated('common.po.goToPeople', 'commmon.po.goToPeopleNative');
```

Comment:
These two methods look the same aside from the deprecated method. Do we need both?